src/components/primitives/Pressable/index.tsx

```diff
@@ -1,2 +1,3 @@
 export { default as Pressable } from './Pressable';
 export type { IPressableProps } from './types';
+export * from './Pressable';
```

Comment:
Please replace `*` with only the hooks you need in `Pressable`. So in future, if we export something it shouldn't get exported with `*`.

Reply:
Thank you for your feedback. I fixed it in the recent commit.